daras_ai_v2/base_v2.py

```
@@ -784,17 +802,141 @@ def _render_about_content(self):
+        if not pr.workspace_id:
+            return
+        workspace = pr.workspace
+
+        count = public_workflow_count(workspace)
+        noun = ngettext(singular="workflow", plural="workflows", number=count)
+        subtitle = f"{format_number_with_suffix(count)} Published {noun}"
+
+        link = self._about_author_href(workspace)
+        with gui.div(className="v2-about-author"):
+            with (
+                link and gui.link(to=link) or gui.dummy(),
```

Ruff (0.16.2)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)